en/dev_log/ulog_file_format.md

```diff
@@ -205,29 +201,28 @@ Predefined information messages are:
 
 key | Description | Example for value
 --- | ---  | ---
-`char[3] sys_name` | Name of the system |  "PX4"
-`char[9] ver_hw` | Hardware version (board) |  "PX4FMU_V4"
-`char[2] ver_hw_subtype` | Board subversion (variation)|  "V2"
-`char[7] ver_sw` | Software version (git tag)|  "7f65e01"
-`char[6] ver_sw_branch` | git branch |  "master"
+`char[value_len] sys_name` | Name of the system |  "PX4"
+`char[value_len] ver_hw` | Hardware version (board) |  "PX4FMU_V4"
+`char[value_len] ver_hw_subtype` | Board subversion (variation)|  "V2"
+`char[value_len] ver_sw` | Software version (git tag)|  "7f65e01"
+`char[value_len] ver_sw_branch` | git branch |  "master"
 `uint32_t ver_sw_release` | Software version (see below)|  0x010401ff
-`char[5] sys_os_name` | Operating System Name |  "Linux"
-`char[7] sys_os_ve`r | OS version (git tag) |  "9f82919"
+`char[value_len] sys_os_name` | Operating System Name |  "Linux"
+`char[value_len] sys_os_ve`r | OS version (git tag) |  "9f82919"
 `uint32_t ver_os_release` | OS version (see below) |  0x010401ff
-`char[7] sys_toolchain` | Toolchain Name |  "GNU GCC"
-`char[5] sys_toolchain_ver` | Toolchain Version |  "6.2.1"
-`char[16] sys_mcu` | Chip name and revision |  "STM32F42x, rev A"
-`char[12] sys_uuid` | Unique identifier for vehicle (eg. MCU ID) |  "392a93e32fa3"...
-`char[7] log_type` | Type of the log (full log if not specified) | "mission"
-`char[10] replay` | File name of replayed log if in replay mode | "log001.ulg"
+`char[value_len] sys_toolchain` | Toolchain Name |  "GNU GCC"
+`char[value_len] sys_toolchain_ver` | Toolchain Version |  "6.2.1"
+`char[value_len] sys_mcu` | Chip name and revision |  "STM32F42x, rev A"
+`char[value_len] sys_uuid` | Unique identifier for vehicle (eg. MCU ID) |  "392a93e32fa3"...
+`char[value_len] log_type` | Type of the log (full log if not specified) | "mission"
+`char[value_len] replay` | File name of replayed log if in replay mode | "log001.ulg"
 `int32_t time_ref_utc` | UTC Time offset in seconds | -3600 |
 
 :::note
-The `key_len` value in `char key[key_len]` has no fixed value.
-Therefore the key lengths in the example above (3, 9, 2, 7, ...) may not be exactly same with your ULog's information message.
+`value_len` represents the data size of the `value`. This is described in the `key`.
 :::
 
-- The format of `ver_sw_release` and `ver_os_release` is: 0xAABBCCTT, where AA is **major**, BB is **minor**, CC is patch and TT is the **type**.
+- For PX4, the format of `ver_sw_release` and `ver_os_release` is: 0xAABBCCTT, where AA is **major**, BB is **minor**, CC is patch and TT is the **type**.
```

Comment:
This does not only apply to PX4, please keep as it was

Reply:
Revert
```suggestion
- The format of `ver_sw_release` and `ver_os_release` is: 0xAABBCCTT, where AA is **major**, BB is **minor**, CC is patch and TT is the **type**.
```